### PR TITLE
Merge Voucher v2.5.2 into grafeas/Voucher

### PR DESCRIPTION
**Pull request by catherinejones**
This PR, assuming I've done it correctly, will merge the changes added in Voucher v2.5.2 to the grafeas version of Voucher.

Thes changes include:

* support for multiple image projects
* better metrics

### CODE REVIEW

**Comment by ooq** [comment]
@catherinejones Can you sign the CLA to unblock the bot?

File: containeranalysis/client.go [hunk 1]
```diff
@@ -13,6 +13,7 @@ import (
 
 	"github.com/Shopify/voucher"
 	"github.com/Shopify/voucher/attestation"
+	"github.com/Shopify/voucher/docker/uri"
```
Comment (ooq) [inline containeranalysis/client.go:16]: s/Shopify/grafeas/g

File: testing/auth.go [hunk 2]
```diff
@@ -10,6 +10,7 @@ import (
 	"golang.org/x/oauth2"
 
 	"github.com/Shopify/voucher"
+	"github.com/Shopify/voucher/auth"
```
Comment (ooq) [inline testing/auth.go:13]: s/Shopify/grafeas/g

File: containeranalysis/poll.go [hunk 3]
```diff
@@ -12,6 +12,7 @@ import (
 	grafeas "google.golang.org/genproto/googleapis/grafeas/v1"
 
 	"github.com/Shopify/voucher"
+	"github.com/Shopify/voucher/docker/uri"
```
Comment (ooq) [inline containeranalysis/poll.go:15]: s/Shopify/grafeas/g

**Review by ooq — APPROVED** [review]
I guess all conflicts are due to renaming Shopify->grafeas. LGTM otherwise.

**Comment by catherinejones** [comment]
@googlebot I fixed it.

**Comment by ooq** [comment]
From the info page, it shows that we are missing CLA from @dani-santos-code to pass this check.
@catherinejones Can you pass the request?

**Comment by dani-santos-code** [comment]
I signed now!

**Comment by ooq** [comment]
@dani-santos-code Thanks! Looks good now.

**Comment by catherinejones** [comment]
This should be covered by this change: https://github.com/grafeas/voucher/pull/1

(It's the same change but in reverse!)

**Pull request closed without merging**